docs/guides/es2k/installing-acc-sdk.md

```diff
@@ -0,0 +1,98 @@
+# Installing the ACC SDK
+
+## Overview
+
+The AARCH64 Software Development Kit (SDK) allows you to use an X86 host
+computer to build P4 Control Plane for use on the ARM Compute Complex
+(ACC) of the Intel&reg; IPU E2100.
+
+The SDK includes the following:
+
+1. GCC C/C++ toolchain. Runs on an X86-64 Linux host and generates binary
+   executables and libraries for an aarch64 target platform.
```

Comment:
AARCH64 in upper case, keep the usage consistent. Previous usage is on line 5. Wikipedia lists it as `AArch64` (https://en.wikipedia.org/wiki/AArch64). I am okay with any option as long it is consistent.

Reply:
Should we mention Rocky Linux AARCH64 target platform here since SDK will be different if ACC is running Debian OS

Reply:
It makes sense to mention Rocky Linux, especially since it's (1) "standard", and (2) embedded in the archive name.